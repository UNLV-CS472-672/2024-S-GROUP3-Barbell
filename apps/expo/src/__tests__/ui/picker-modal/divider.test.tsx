import React from 'react'

import { render } from '@testing-library/react-native'

import Divider from '~/components/ui/picker-modal/divider/divider'

describe('Divider', () => {
  it('renders correctly', () => {
    const { getByTestId } = render(<Divider />)
    expect(getByTestId('divider')).toBeTruthy()
  })

  it('applies custom styles', () => {
    const customStyle = { backgroundColor: 'blue' }
    const { getByTestId } = render(<Divider style={customStyle} />)
    expect(getByTestId('divider')).toHaveStyle(customStyle)
  })
})
