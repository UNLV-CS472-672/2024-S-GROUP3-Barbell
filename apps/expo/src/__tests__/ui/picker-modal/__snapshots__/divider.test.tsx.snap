// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider matches snapshot 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#3b3b3b",
        "height": 1,
        "width": "100%",
      },
      undefined,
    ]
  }
  testID="divider"
/>
`;
